data-catalog-backend-app/src/main/java/no/nav/data/catalog/backend/app/codelist/CodelistService.java

```diff
@@ -40,18 +41,33 @@ public void refreshCache() {
 		});
 	}
 
-	public Codelist save(CodelistRequest request) {
-		codelists.get(request.getList()).put(request.getCode(), request.getDescription());
-		return repository.save(request.convert());
+	public List<Codelist> save(List<CodelistRequest> requests) {
+		requests.forEach(request -> codelists.get(request.getList())
+				.put(request.getCode().toUpperCase(), request.getDescription()));
+		return repository.saveAll(requests.stream()
+				.map(CodelistRequest::convert)
+				.collect(Collectors.toList()));
 	}
 
-	public Codelist update(CodelistRequest request) {
-		codelists.get(request.getList()).put(request.getCode(), request.getDescription());
+	public List<Codelist> update(List<CodelistRequest> requests) {
+		requests.forEach(request -> codelists.get(request.getList())
+				.put(request.getCode().toUpperCase(), request.getDescription()));
 
-		Codelist toBeUpdatedCodelist = repository.findByListAndCode(request.getList(), request.getCode()).get();
-		toBeUpdatedCodelist.setDescription(request.getDescription());
+		return repository.saveAll(requests.stream()
+				.map(this::setDescriptionByCodelistRequest)
+				.collect(Collectors.toList()));
+	}
 
-		return repository.save(toBeUpdatedCodelist);
+	private Codelist setDescriptionByCodelistRequest(CodelistRequest request) {
+		Optional<Codelist> optionalCodelist = repository.findByListAndCode(request.getList(), request.getCode().toUpperCase());
```

Comment:
Check for null values and add them error before finding.

Reply:
Null values will be validated before the request get here.
Validation is commented out until branch #42